### ISSUE TITLE
Cookie Pop-up filter not work for Firefox

### ISSUE THREAD

**Issue by Morku** [AdguardTeam/AdguardBrowserExtension#2640]
### AdGuard Extension version

4.2.228

### Browser version

Firefox 115.5.0esr

### OS version

Windows 10 21H2 Build 19044.3693

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard German filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Annoyances filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Widgets filter, AdGuard Other Annoyances filter

### What Stealth Mode options do you have enabled?

Block trackers, Remove tracking parameters, Hide your search queries, Send websites signals not to track you, Remove X-Client-Data header

### Issue Details

The YouTube Cooke Pop-up does not get skipped with AdGuard Extension on Firefox. Using AdGuards Cookie filter.

- When use AdGuard Extension on a Chromium based browser (Edge, Brave) -> the Cookie Pop-up gets autoclicked without issue -> no filter issue
- When I use AdGuard for Windows and Firefox -> the Cookie Pop-up also gets autoclicked without issue -> no filter issue
- When I use uBlock Origin -> the Cookie Pop-up also gets autoskipped (even when they have another syntax) -> no Firefox limitation

So I have to conclude that it comes down to the AdGuard Extension, being limited with Firefox.

### Expected Behavior

Having the same filter quality like with a Chromium based browser. I doubt that this is a Firefox limitation, as it feels like every source says, that Firefox Add-Ons are less restricted than on Chromium. Or to quote uBlock: "uBlock Origin works best on Firefox"

### Screenshots

https://github.com/AdguardTeam/AdguardBrowserExtension/assets/6276915/69d2255a-b7a0-414c-af04-653c2325ccb7

**Comment by windwak3r** [comment]
@Morku Is the problem still relevant?

**Comment by Morku** [comment]
@windwak3r 
No, the extension now works as expected on Firefox. You can close the ticket.
